src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -80,12 +78,11 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
   } = useFieldArray({ control, name: 'constraints' });
   const { mutateAsync } = useEditSingleProject(project.wbsNum);
 
-  if (allUsers.isLoading || !allUsers.data || !auth.user) return <LoadingIndicator />;
+  if (allUsers.isLoading || !allUsers.data) return <LoadingIndicator />;
   if (allUsers.isError) {
     return <ErrorPage message={allUsers.error?.message} />;
   }
 
-  const { userId } = auth.user;
   const users = allUsers.data.filter((u) => u.role !== 'GUEST');
 
   const onSubmit = async (data: any) => {
```

Comment:
Probably shouldn't use `any`.

Reply:
true. i might turn that into a bunch of tickets to fix how we do all these payloads since i hate how all of them are just fat `any` objects

Reply:
I made a ticket for it already.